execution/inc/leatherman/execution/execution.hpp

```diff
@@ -430,6 +434,7 @@ namespace leatherman { namespace execution {
      */
     std::tuple<std::string, std::string> process_streams(
             bool trim,
+            bool strip_cr,
```

Comment:
On a related note, https://github.com/puppetlabs/facter/blob/master/lib/inc/internal/execution/execution.hpp looks like it should've been removed (it's not used anywhere). I'll throw up a PR.